src/main/java/com/google/api/codegen/viewmodel/testing/MockGrpcMethodView.java

```diff
@@ -25,6 +25,8 @@
 
   public abstract String responseTypeName();
 
+  public abstract String streamHandle();
```

Comment:
I left it rather vague because I didn't know what name to generalize it with. In Go, this is the type name of the "stream object", which you can use to send and receive data. An analog in Java would probably be `StreamObserver` though Go has both send and receive on the same object.

streamHandleTypeName?